src/app/activity/bounty/Publish.ts

```diff
@@ -29,8 +29,8 @@ export const publishBounty = async (publishRequest: PublishRequest): Promise<any
 	const bountyChannel: TextChannel = dbBountyResult.createdInChannel ? await DiscordUtils.getTextChannelfromChannelId(dbBountyResult.createdInChannel) :
 		 await guildMember.client.channels.fetch(dbCustomerResult.bountyChannel) as TextChannel;
 	const bountyMessage: Message = await BountyUtils.canonicalCard(dbBountyResult._id, publishRequest.activity, bountyChannel);
-    await guildMember.send({ content: `Bounty published to ${bountyChannel.name} <${bountyMessage.url}> and the website! <${process.env.BOUNTY_BOARD_URL}${bountyId}>` });
-	Log.info(`bounty published to ${bountyChannel.name}`);
+    await guildMember.send({ content: `Bounty published to \`${(bountyMessage.channel as any).name || bountyChannel.name}\` <${bountyMessage.url}> and the website! <${process.env.BOUNTY_BOARD_URL}${bountyId}>` });
```

Comment:
Here if you had to use the default channel, let the user know as part of this message how they can remedy that, like sesh does.